manual/python/index.md

```diff
@@ -60,9 +60,87 @@ caption="Example of graphics generated with PyROOT."
+
+```python
+import ROOT
+ROOT.PyConfig.OptionName1 = SomeValue1
+ROOT.PyConfig.OptionName2 = SomeValue2
+```
+
+The available configuration options are described next:
+
+- `DisableRootLogon` (default `False`): just like it happens in C++, PyROOT also supports the [rootlogon](https://root.cern/doc/master/rootlogon_8C.html) functionality. When PyROOT starts, it will look for a file called `.rootlogon.py` in the user's home directory and, if such file exists, PyROOT will import it. This can be used to make some settings every time PyROOT is launched, for example defining some style for your plots:
+
+```python
+# Content of .rootlogon.py
+import ROOT
+myStyle = ROOT.TStyle('MyStyle','My graphics style')
+myStyle.SetCanvasColor(ROOT.kBlue) # My canvases will have blue color!
+ROOT.gROOT.SetStyle('MyStyle')
+```
+
+If PyROOT cannot find `.rootlogon.py` in the user's home directory, it will look for the equivalent in C++ (`.rootlogon.C`), first in [ROOT's etc directory](https://root.cern/doc/master/classTROOT.html#ab8e51627a12d886d6c8177b46481352a), then in the user's home directory and finally in the current working directory. Note that it is also possible to use both the Python and the C++ rootlogons, since the latter can be loaded from the former, for instance with `ROOT.gROOT.LoadMacro('.rootlogon.C')`.
+
+If the user would like to completely disable the rootlogon functionality, they can do so by setting `PyConfig.DisableRootLogon` to `True`. 
+
+- `IgnoreCommandLineOptions` (default `True`): if a PyROOT script is executed with some command line arguments, they will be ignored by default by ROOT, so the user is free to process them as they wish. However, by setting `PyConfig.IgnoreCommandLineOptions` to `False`, those arguments will be forwarded to ROOT for parsing, e.g. to enable the batch mode from the command line. A complete list of the arguments accepted by ROOT can be found [here]({{ '/manual/first_steps_with_root/#starting-root-with-command-line-options' | relative_url }}).
+
+- `ShutDown` (default `True`): when PyROOT is terminating, during its cleanup phase, the ROOT C++ interpreter is shut down. If PyROOT is executed as part of a longer-running application that needs the C++ interpreter, `PyConfig.Shutdown` can be set to `False` to prevent that shutdown.
+
+- `StartGUIThread` (default `True`): when not executing in [interactive mode](https://docs.python.org/3/tutorial/interpreter.html#interactive-mode), PyROOT starts a thread that periodically polls for ROOT events (e.g. GUI events) to process them. If a given PyROOT application does not need this event processing, it can prevent the creation of the thread by setting `PyConfig.StartGUIThread` to `False`.
```

Comment:
Interactive mode or from a Jupyter notebook would be technically correct. I don't know, is it important?

Reply:
Jupyter notebook falls in the IPython category, so I can say `interactive mode or IPython/Jupyter` ?